content/blog/2019-10-21-rmangal.md

```diff
@@ -0,0 +1,154 @@
+theoreticians, that can test ecological theory with such datasets.
+
+We had two main objectifs for [rmangal 2.0.0][rmangal]. First, the rmangal
+package had to allow users to search for all entries in the database in a very
+flexible way. From a technical point this means that we had to write functions
+to query all the endpoints of the [new web API][API]. The second goal was to
+make the package as user friendly as possible. To do so, we used explicit and
+consistent names for functions and arguments, designed a simple workflow, and
+documented how to use other field related packages (such as [igraph][igraph]) to
+visualize and analyze networks. All of this is explain in the vignette ["get
+started with rmangal"](ttps://ropensci.github.io/rmangal/articles/rmangal.html).
+
+
+
+### A successful peer-review process
+
+<!-- version to prerelease -->
+After some hard work behind the screen and once we deemed our two objectifs
+achieved, we decided to submit rmangal package to rOpenSci for peer-review. We
+did so because we needed feedback, we needed qualified people to critically
+assess whether our two main objectifs were achieved. Given the strong expertise
+of rOpenSci in software review, and given that our package was in-scope,
+submitting rmangal to rOpenSci was an obvious choice.
+
+We [had very valuable
+feedback](https://github.com/ropensci/software-review/issues/332) from [Anna
+Willoughby][arw36] and [Thomas Lin Pedersen][thomasp85]. They carefully assessed
+our work and pointed out areas where improvement was required. One good example
+of how their review made our package better concerns the dependencies. We
+originally listed [sf][sf] in `Imports` as we used it to filter networks based
+on geographic coordinates. But the reviewers pointed out that this was not an
+essential part of the package and that [sf][sf] has several dependencies. This  made us realize that for one extra feature, we were substantially
+increasing the number of indirect dependencies. Following the reviewers'
+suggestions, we moved [sf][sf] to `Suggests` and added a message to warn users
+that to use the [spatial filtering
+feature](https://ropensci.github.io/rmangal/articles/rmangal.html#geolocalize-mangal-networks-with-sf)
+requires sf to be installed. Similarly, based on another good comment, we
+added a function to convert Mangal networks into [tidygraph][tidygraph] objects
+and we documented how to plot Mangal networks with [ggraph][ggraph] (and so we
+added those packages in `Suggests`). Such improvements were very helpful to
+properly connect rmangal to the existing R packages. The plethora of R packages
+is one of its major strength, and connecting a package properly to other makes
+the entire ecosystem even stronger.
+
+<!-- Future -->
+We are now looking for user feedback, not only for rmangal but also for the web
+API and the website. We welcome suggestions and contributions, especially for
```

Comment:
Do you have any specific areas in which you'd like feedback? How do you prefer people contact you for this e.g. open an issue?

This whole post reads well, so I don't want to mess with it, but giving some specifics here if possible, might make the difference in people actually taking a step to contribute.

Reply:
Hum, I think it is a good point we were thinking about posting issue for feedback, I'll add something!

Reply:
:+1:

Reply:
Changes in 3487f1825a3ed7876e77e84a4c8a0421c048da02